chia/util/cached_bls.py

```diff
@@ -42,12 +42,12 @@ def get_pairings(
                 pk_parsed = G1Element.from_bytes(pks[i])
                 pk_bytes_to_g1[pks[i]] = pk_parsed
 
-            pairing = pk_parsed.pair(aug_hash)
+            pairing = aug_hash.pair(pk_parsed)
 
             h = std_hash(aug_msg)
             cache.put(h, pairing)
             pairings[i] = pairing
-    return pairings
+    return cast(List[GTElement], pairings)
```

Comment:
Maybe it would be better to have a separate list hinted properly such that mypy can help us enforce this?

Reply:
I went for `cast` to avoid making changes to the original intention. But I'll add a separate list to make the types more explicit